### ISSUE TITLE
Rename "Fixed Output" models

### ISSUE THREAD

**Issue by joernmalzahn** [geez0x1/CompliantJointToolbox#55]
The name "Fixed Output" is misleading. The models capture an actuator that connected to something *very heavy*, that would instantaneously create a reaction torque to any actuator effort passing through the elastic element. The actuator is however not literally **fixed** as it was rigidly connected to a wall. 
This way, it can be used to impose an output motion, that the actuator can only react to (through the second input ). 

The case of the actuator being literally **fixed** to a wall is only the special case for $\dot{q}_l = 0$. The models can do much more than just that.

Changing the names of these models will have substantial impact on the toolbox code in many places. So I mark this as an enhancement. First we should find a better name. Let's discuss this here.

**Comment by geez0x1** [comment]
I agree that the terminology is misleading and that it should be clarified. The main function of these models is to allow connection of the actuator model to other dynamic models, e.g. complex articulated robot dynamics, other actuators, specific motion profiles (e.g. to simulate an assistive device where the human imposes the motion for the most part), and also the "infinitely heavy" case where q_l_dot=0.

Hence "very heavy" isn't exactly correct, nor is "output_fixed" precise enough. I think the new names should focus on the fact that together with physical torque output (TorqueNoNoise, see #47), the load motion input creates an interaction port, exchanging mechanical power between the two systems.
